checklist.md

```diff
@@ -8,6 +8,6 @@ Let's create `add(Order order)` instead.
 * Check carefully the relationships in `Order` class. You should use `OneToMany` with `tickets`.
 * When passing tickets from shoppingCart to order - pass the copy of this list(you can use `ArrayList` constructor for this);
 The reason is that Hibernate will automatically delete these tickets from shoppingCart(in DB) after noticing them in our order object. 
-Let's do this manually using `clear()` method from `ShoppingCartService` after the order with tickets has been created.
+Let's do this manually using `clearShoppingCart()` method from `ShoppingCartService` after the order with tickets has been created.
```

Comment:
be careful with project-wide replacements, review your commits to avoid unintended changes